server/config/middleware.js

```diff
@@ -1,3 +1,21 @@
+var memeController = require('../memes/memeController.js');
+
 module.exports = function (app, express) {
+  // Express 4 allows us to use multiple routers with their own configurations
+  var memeRouter = express.Router();
+
   app.use(express.static(__dirname + '/../../client'));
+
+  app.get('/poo', function(req, res){
+    res.send('hello index');
+  });
+
+  app.get('/memes', memeController.getMemes);
+  app.get('/memes/create', memeController.createMeme);
```

Comment:
Probably, but at the moment app.post() breaks the code.  Will have to check into this further!